### PR TITLE
new extension nanoid

### PR DESCRIPTION
**Pull request by roelvan**
## Description

New extension to generate [nanoid](https://github.com/ai/nanoid)'s.

## Screencast

Video: https://share.cleanshot.com/ra57LS

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/nanoid/package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  "icon": "command-icon.png",
+  "author": "roelvan",
+  "categories": [
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "Nanoid",
+      "title": "Nanoid",
```
Comment (mattisssa) [inline extensions/nanoid/package.json:15]:
```suggestion
      "title": "Generate Nanoid",
```

File: extensions/nanoid/package.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  "description": "A tiny (130 bytes), secure, URL-friendly, unique string ID generator for JavaScript ",
+  "icon": "command-icon.png",
+  "author": "roelvan",
+  "categories": [
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "Nanoid",
```
Comment (mattisssa) [inline extensions/nanoid/package.json:14]:
```suggestion
      "name": "nanoid",
```

File: extensions/nanoid/package.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "nanoid",
+  "title": "nanoid",
+  "description": "A tiny (130 bytes), secure, URL-friendly, unique string ID generator for JavaScript ",
```
Comment (mattisssa) [inline extensions/nanoid/package.json:5]:
```suggestion
  "description": "A tiny, secure, URL-friendly, unique string ID generator for JavaScript ",
```

**Review by mattisssa — CHANGES_REQUESTED** [review]
Hey, thanks for the contribution! Nice icon 😉 

I guess the advantage of your extension comparing with [this one](https://www.raycast.com/loris/random) is that it can be assigned with shortcut and supporting just a single nanoid data? Because users could do the same with extension I linked (with 1 more step then with your extension)

I also suggested a few changes that must be done to pass the CI validation and to match the store guidelines.

**Review by roelvan — COMMENTED** [review]
The faker extension you mention cannot generate nanoid's afaik.

**Comment by mattisssa** [comment]
> The faker extension you mention cannot generate nanoid's afaik.

alright 👌 
I thought UUID is kinda the same thing.

**Pull request merged**